spec/models/post_spect.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] Kindly add model specs, this means that we should test the creation of a post with valid and invalid attributes.
- [ ] Let's rename our files to end with `spec` instead of `spect`